python/py_helpers.py

```diff
@@ -99,6 +99,11 @@ def get_variable(self, name):
     def has_function(self, name):
         return self.find_function(name) != Node()
 
+    # Checks if the current scope contains a "pass" statement
+    
+    def has_pass(self):
+        return any(isinstance(node, ast.Pass) for node in self.tree.body)
```

Comment:
This raises an exception if `self.tree.body` is missing, right? 

If that's correct, how about just returning false if there isn't a body?

Reply:
```suggestion
        try:
            return any(isinstance(node, ast.Pass) for node in self.tree.body)
        except AttributeError:
            return False
```
Is something like this acceptable? @ojeytonwilliams

Reply:
```suggestion
        if getattr(self.tree, 'body', False):
            return any(isinstance(node, ast.Pass) for node in self.tree.body)
        return False
```
Probably better...